commit b81afe37ff5598d7bbb5efa5f2ef6997a5d48e5e
Author: Gustavo A. R. Silva <gustavo@embeddedor.com>
Date:   Fri Jan 25 15:55:33 2019 -0600

    drm/amd/powerplay: Fix missing break in switch
    
    commit 2f10d823739680d2477ce34437e8a08a53117f40 upstream.
    
    Add missing break statement in order to prevent the code from falling
    through to the default case.
    
    The resoning for this is that pclk_vol_table is an automatic variable.
    So, it makes no sense to update it just before falling through to the
    default case and return -EINVAL.
    
    This bug was found thanks to the ongoing efforts to enabling
    -Wimplicit-fallthrough.
    
    Fixes: cd70f3d6e3fa ("drm/amd/powerplay: PP/DAL interface changes for dynamic clock switch")
    Cc: stable@vger.kernel.org
    Signed-off-by: Gustavo A. R. Silva <gustavo@embeddedor.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/smu10_hwmgr.c b/drivers/gpu/drm/amd/powerplay/hwmgr/smu10_hwmgr.c
index a63e00653324..1546bc49004f 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/smu10_hwmgr.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/smu10_hwmgr.c
@@ -984,6 +984,7 @@ static int smu10_get_clock_by_type_with_latency(struct pp_hwmgr *hwmgr,
 		break;
 	case amd_pp_dpp_clock:
 		pclk_vol_table = pinfo->vdd_dep_on_dppclk;
+		break;
 	default:
 		return -EINVAL;
 	}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1339_b81afe37ff5598d7bbb5efa5f2ef6997a5d48e5e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1339_b81afe37ff5598d7bbb5efa5f2ef6997a5d48e5e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
